Gemfile

```diff
@@ -7,9 +7,9 @@ gem 'nokogiri'
 
 ### Fastlane Plugins
 
-gem 'fastlane-plugin-wpmreleasetoolkit', '~> 6.0'
+# gem 'fastlane-plugin-wpmreleasetoolkit', '~> 6.0'
 #gem 'fastlane-plugin-wpmreleasetoolkit', path: '../../release-toolkit'
-#gem 'fastlane-plugin-wpmreleasetoolkit', git: 'https://github.com/wordpress-mobile/release-toolkit', branch: '…'
+gem 'fastlane-plugin-wpmreleasetoolkit', git: 'https://github.com/wordpress-mobile/release-toolkit', branch: 'firebase-test-return-url'
```

Comment:
TODO: revert this and point to the new version of the `release-toolkit` once https://github.com/wordpress-mobile/release-toolkit/pull/430 lands and we published a new release.